source/manual/sidekiq.html.md

```diff
@@ -44,29 +44,16 @@ There are three approaches for monitoring Sidekiq:
 
 ### Sidekiq Web (aka Sidekiq Monitoring)
 
-Sidekiq comes with a web application,
-[`Sidekiq::Web`](https://github.com/mperham/sidekiq/wiki/Monitoring)
-that can display the current state of a Sidekiq installation. We have
-configured this as the [sidekiq-monitoring web app](https://github.com/alphagov/sidekiq-monitoring)
-to monitor multiple Sidekiq configurations used throughout GOV.UK.
-
-We have restricted public access because the Web UI allows modifying the
-state of Sidekiq queues. To access the Sidekiq monitoring UI for
-Production AWS, run:
-
-```bash
-$ gds govuk connect sidekiq-monitoring -e production aws/backend
-```
-
-The output will be something like:
+Sidekiq comes with a web application, [`Sidekiq::Web`][sidekiq web] that can
+display the current state of Sidekiq's queues for an application. We have
+configured this as the [sidekiq-monitoring web app][sidekiq monitoring] which
+can monitor multiple Sidekiq configurations used throughout GOV.UK.
 
-```
-Port forwarding setup, access:
-
-  http://127.0.0.1:xxxxx/
-```
+You can access each applications dashboard from [this URL][dashboard].
```

Comment:
Do we need links here for each environment?

Reply:
We do indeed. Is there a fancy helper for this?

Reply:
I've just raised [a PR](https://github.com/alphagov/govuk-developer-docs/pull/3024) for a helper to render links to Jenkins and we already have `RunRakeTask`.  Maybe these two could be consolidated into a new helper that can produce links to other things too?

Reply:
Ooh, good idea! We do have variations of URL's for individual applications per environment which may make auto-generating them a bit tricky, e.g:

* https://grafana.blue.integration.govuk.digital/
* https://graphite.integration.govuk.digital/
* https://content-data.integration.publishing.service.gov.uk/

Reply:
I'll merge this PR in with the manual links and have a ponder.